pkg/local_object_storage/engine/engine_test.go

```diff
@@ -55,13 +56,13 @@ func testNewShard(t *testing.T, id int) *shard.Shard {
 		shard.WithID(sid),
 		shard.WithLogger(zap.L()),
 		shard.WithBlobStorOptions(
-			blobstor.WithRootPath(fmt.Sprintf("%s.%d.blobstor", t.Name(), id)),
+			blobstor.WithRootPath(path.Join(t.Name(), fmt.Sprintf("%d.blobstor", id))),
```

Comment:
What's the problem with the old scheme?

Reply:
Each test creates 2 blobovnicza dirs and 2 metabase files directly in the engine dir.
```
$ tree
./engine
├── TestListWithCursor.1.blobstor
│   └── blobovnicza
├── TestListWithCursor.1.metabase
├── TestListWithCursor.2.blobstor
│   └── blobovnicza
└── TestListWithCursor.2.metabase
```

It is easier to cleanup files I they are placed in separate dir.
```go
	t.Cleanup(func() {
		e.Close()
		os.RemoveAll(t.Name())
	})
```
```
./engine
└── TestListWithCursor
    ├── 1.blobstor
    │   └── blobovnicza
    ├── 1.metabase
    ├── 2.blobstor
    │   └── blobovnicza
    └── 2.metabase
```